### PR TITLE
add doc strings to all launch file args.

### PR DESCRIPTION
**Pull request by hsd-dev**
Using the 'doc' attribute on 'arg' elements.

### CODE REVIEW

File: motoman_driver/launch/motion_streaming_interface.launch [hunk 1]
```diff
@@ -3,10 +3,10 @@
 -->
 <launch>
 	<!-- IP of robot (or PC running simulation) -->
-	<arg name="robot_ip" />
+	<arg name="robot_ip" doc="IP of controller (only required if not using industrial simulator)" />
```
Comment (gavanderhoorn) [inline motoman_driver/launch/motion_streaming_interface.launch]: This launch file cannot be used with the `industrial_robot_simulator`, so this seems like a strange comment as a doc string.

Can you clarify?

File: motoman_mh5_support/launch/robot_interface_streaming_mh5.launch [hunk 2]
```diff
@@ -9,10 +9,10 @@
     robot_interface_streaming_mh5.launch robot_ip:=<value> controller:=<fs100|dx100>
 -->
 <launch>
-	<arg name="robot_ip" />
+	<arg name="robot_ip" doc="IP of controller (only required if not using industrial simulator)" />
 	
   <!-- controller: Controller name (fs100 or dx100) -->
-  <arg name="controller"/>
+  <arg name="controller" doc="Name of the controller"/>
```
Comment (gavanderhoorn) [inline motoman_mh5_support/launch/robot_interface_streaming_mh5.launch]: This is a bit ambiguous: it's really the controller *series* or *type* that is specified by the `controller` `arg`.

Comment (gavanderhoorn) [reply motoman_mh5_support/launch/robot_interface_streaming_mh5.launch]: Would probably also be good to add the possible values there (ie: `dx100`, `fs100`, `dx200` and `yrc1000`).

File: motoman_sda10f_moveit_config/launch/default_warehouse_db.launch [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <launch>
 
-  <arg name="reset" default="false"/>
+  <arg name="reset" default="false" doc="Reset DB"/>
```
Comment (gavanderhoorn) [inline motoman_sda10f_moveit_config/launch/default_warehouse_db.launch]: This file (and all others in MoveIt configurations) will all be overwritten if the Setup Assistant is ever run again.

Do you feel it's worth it to manually add doc strings here, instead of updating the MSA templates upstream?

File: motoman_driver/launch/motion_streaming_interface.launch (hunk 1 above)
Comment (gavanderhoorn) [reply motoman_driver/launch/motion_streaming_interface.launch]: Additionally: the `arg` is really a required arg (there is no `default` attribute), so it's always required.

**Review by gavanderhoorn — CHANGES_REQUESTED** [review]
Thanks for the PR. I've added some comments.

File: motoman_mh5_support/launch/robot_interface_streaming_mh5.launch (hunk 2 above)
Comment (hsd-dev) [reply motoman_mh5_support/launch/robot_interface_streaming_mh5.launch]: Modified.

File: motoman_sda10f_moveit_config/launch/default_warehouse_db.launch (hunk 3 above)
Comment (hsd-dev) [reply motoman_sda10f_moveit_config/launch/default_warehouse_db.launch]: Wasn't quite sure about this, reverted back.

On a side note, I couldn't find documentation for args/params used in move_group.launch like `max_safe_path_cost`

File: motoman_driver/launch/motion_streaming_interface.launch (hunk 1 above)
Comment (hsd-dev) [reply motoman_driver/launch/motion_streaming_interface.launch]: Somehow overlooked the missing default. Modified.

File: motoman_sda10f_moveit_config/launch/default_warehouse_db.launch (hunk 3 above)
Comment (gavanderhoorn) [reply motoman_sda10f_moveit_config/launch/default_warehouse_db.launch]: That would be something to take up with the moveit team :)

File: motoman_sda10f_moveit_config/launch/move_group.launch [hunk 4]
```diff
@@ -13,7 +13,7 @@
   <arg     if="$(arg info)" name="command_args" value="--debug" />
 
   <!-- move_group settings -->
-  <arg name="allow_trajectory_execution" default="true"/> 
+  <arg name="allow_trajectory_execution" default="true"/>
```
Comment (gavanderhoorn) [inline motoman_sda10f_moveit_config/launch/move_group.launch]: Just noticed this whitespace change. Could you revert the changes to this file? They are out-of-scope for this PR.

**Comment by gavanderhoorn** [comment]
Thanks for the updated PR. One minor comment left. If you could address that we can merge.

**Review by gavanderhoorn — APPROVED** [review]

**Comment by gavanderhoorn** [comment]
Just waiting on Travis.

**Comment by gavanderhoorn** [comment]
Thanks for the PR @ipa-nhg-hd. 👍

**Comment by hsd-dev** [comment]
you are welcome :)
